lib/common.css

```diff
@@ -242,4 +247,4 @@ paper-dialog input::selection,
 paper-dialog textarea::selection {
   background: <<ALPHA 0.6>><<FORE_SECONDARY>><</ALPHA>>;
 }
-/** Update End **/
```

Comment:
The change appears to be made automatically by your IDE, please reverse / unstage it 👍